packages/gatsby/src/redux/actions/restricted.ts

```diff
@@ -314,6 +314,14 @@ export const actions = {
    * @param {string[]} [$0.exclude.types] Do not include these types
    * @param {string[]} [$0.exclude.plugins] Do not include types owned by these plugins
    * @param {boolean} [withFieldTypes] Include field types, defaults to `true`
+   * @example
+   * exports.createSchemaCustomization = ({ actions }) => {
+   *   actions.printTypeDefinitions({})
+   * }
+   * @example
+   * exports.createSchemaCustomization = ({ actions }) => {
+   *   actions.printTypeDefinitions({ withFieldTypes: false })
```

Comment:
When I ran this second example locally, I didn't actually see any differences in the `schema.gql` file generated.

If there's a particular difference you were hoping to call out in this example, I'd add a comment to the annotation to highlight the difference. Otherwise, I think you can remove it.

Reply:
I added that second example so an user could see an example with arguments. I can remove it if you want.

Reply:
@meganesu?

Reply:
Hi @Kolkies, thanks for your patience. Agreed, I think it makes sense to have an example with arguments. Could you add a comment (similar to the first example) calling out what this line will do? Specifically to call out how the output will be different from the first example.

Then I think this PR will be good to go!

Reply:
I added a comment, can you check it?